lib/sen_tweet/metrics.ex

```diff
@@ -9,7 +9,7 @@ defmodule SenTweet.Metrics do
       tweets_processed: 0,
       sum_scores: 0,
       average_score: 0,
-      histogram: Enum.map(0..10, fn(x) -> [-100 + 200*x/11, -100 + 200*(x+1)/11, 0] end)
+      histogram: Enum.map(0..10, fn(x) -> [-1 + 2*x/11, -1 + 2*(x+1)/11, 0] end)
```

Comment:
Number of bins is crudely hard-coded. Not sure the best way to approach this if you want to make it configurable.  Data is stored as a list of lists: `[start, end, count]` for each bin

Reply:
Does the number of bins need to be configurable?

Reply:
Also, you can use the **capture operator** for this mapping function.

```
Enum.map(0..10, &[-1 + 2*&1/11, -1 + 2*(&1+1)/11, 0])
```

Reply:
Capture operator is pretty cool. I remember mathematica has a feature like this.. they call it a "pure function".

Regarding the number of bins, I don't know if it needs to be easily configurable. Maybe we could get it to update automatically.. usually when you have more data, you want more bins so you can see finer structure in the distribution. Lots of bins with little data just looks like noise, though. I'll think on how we might auto-update. I know there are some heuristics out there